src/main/telemetry/frsky.c

```diff
@@ -190,7 +191,14 @@ static void sendThrottleOrBatterySizeAsRpm(void)
 {
     sendDataHead(ID_RPM);
     if (ARMING_FLAG(ARMED)) {
-        serialize16(rcCommand[THROTTLE] / BLADE_NUMBER_DIVIDER);
+        if(rcCommand[THROTTLE] <= escAndServoConfig->minthrottle){
```

Comment:
https://github.com/cleanflight/cleanflight/blob/master/src/main/flight/mixer.c#L688 condition and values should be probably used if you insist on sending throttle this way ...